src/ducks/subject.js

```diff
@@ -270,12 +260,7 @@ const fetchSubject = (initialFetch = false) => {
         });
     };
 
-    if (checkEmergencySave(getState().login.user)) {  //If not, check if there's an emergency save.
-      dispatch(emergencyLoadWorkInProgress());
-      dispatch(toggleDialog(<DialogOfContinuation dispatch={dispatch} />, false, false));
-    } else if (savedClassificationPrompt) {  //Check if the user has manually saved progress. (Emergency save trumps manual save.)
-      dispatch(toggleDialog(<ClassificationPrompt />, false, true));
-    } else if (!getState().subject.queue.length) {  //If not, check if there are any subjects left in the queue.
+    if (!getState().subject.queue.length) {  //If not, check if there are any subjects left in the queue.
```

Comment:
Can you confirm my assumptions here. This logic has been changed here (removing emergency save checks) because this functionality is now covered by the classification page?